tests/test_pe_reports.py

```diff
@@ -0,0 +1,33 @@
+"""File to be used to create test for pe-reports."""
+
+# Standard Python Libraries
+import unittest
+
+
+class TestingStringMethods(unittest.TestCase):
+    """Simple test for first-commit only."""
```

Comment:
Since you created dummy tests here, please create an issue in this repo to replace these tests with useful tests.  Then add a `TODO` comment somewhere in this file with a brief note and a link to the issue you created.

Reply:
Addressed in 8806233

Reply:
Thank you for creating the issue, though in the future, please fill out our issue template instead of overwriting it with a copy/paste of my comment.

Reply:
Got it!